portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/blogs/BlogsEntry.testcase

```diff
@@ -145,13 +145,32 @@ definition {
 	}
 
 	@description = "This is a test for LPS-136763. It checks that 400 characters can be added to the description."
-	@ignore = "true"
 	@priority = "3"
 	test CanAddDescriptionWith400Characters {
-		property portal.acceptance = "false";
+		JSONLayout.addPublicLayout(
+			groupName = "Guest",
+			layoutName = "Blogs Page");
+
+		JSONLayout.addWidgetToPublicLayout(
+			column = "1",
+			groupName = "Guest",
+			layoutName = "Blogs Page",
+			widgetName = "Blogs");
 
-		// TODO LPS-136763 CanAddDescriptionWith400Characters pending implementation
+		Navigator.gotoPage(pageName = "Blogs Page");
+
+		Blogs.addEntry(
+			entryContent = "|||||||||1|||||||||2|||||||||3|||||||||4|||||||||5|||||||||6|||||||||7|||||||||8|||||||||9||||||||10||||||||11||||||||12||||||||13||||||||14||||||||15||||||||16||||||||17||||||||18||||||||19||||||||20||||||||21||||||||22||||||||23||||||||24||||||||25||||||||26||||||||27||||||||28||||||||29||||||||30||||||||31||||||||32||||||||33||||||||34||||||||35||||||||36||||||||37||||||||38||||||||39||||||||40",
```

Comment:
Hi @LeandroTravassos, the description usually refers to custom abstraction. See step#5 that Austin mentioned [here](https://issues.liferay.com/browse/LPS-136763?focusedCommentId=2537919&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2537919). So you need to adjust the test.

You can close this PR and resend a new one with the changes. Thanks.